### DISCUSSION TITLE
Changelog?

### DISCUSSION THREAD

**Discussion by PagesplitPeter** [grate-devs/grate#574] [General]
We are currently using Grate 1.6 and I was looking for what has changed in 1.8 but I can't find it. Either I'm looking in the wrong places or its well hidden :) 

Is there a changelog somewhere that has a short description of what has been changed between versions? Like what bugs has been fixed, new improvements and new features?

If this exists somewhere can it be added to the grate home page in some way?

If it doesn't exists is this something that could be considered to be added to the project. It can just be a markdown file in the repo :)

**Comment by jswright91** [comment]
you can find them on the releases page: https://github.com/erikbra/grate/releases

**Reply by PagesplitPeter** [reply]
Thanks!!, might be a good idea to have a link on the webpage to them :)

**Reply by jswright91** [reply]
There's a link on the homepage of the git repo.
![image](https://github.com/user-attachments/assets/08ace70a-b3a4-48b5-8810-5e790b1fc63a)

I'm not sure a link on the website is necessary, but I'll link @erikbra so he can look at the suggestion.

**Reply by PagesplitPeter** [reply]
It not really necessary, more like good to have.
I had a hard time finding this link you showed me since I dont use github that much. More in azure devops due to work :)

**Comment by PagesplitPeter** [comment]
Thanks for he info closing this
